.github/actions/artifact-size-metrics/utils/download-and-process/compare.sh

```diff
@@ -0,0 +1,50 @@
+# Compares artifact size metrics to ones from the latest available release,
+# stores comparison as a markdown table,
+# and returns "true" if a large diff was found (over 5%)
+compareMetrics() {
+    local metrics_file="$1"
+    local latest_release_metrics_file="$2"
+    local metrics_comparison_file="$3"
+
+    # Title and table headers
+    {
+      echo "Affected Artifacts"
+      echo "="
+      echo "| Artifact | Pull Request (bytes) | Latest Release (bytes) | Delta (bytes) | Delta (percentage) |"
+      echo "|----------|----------------------|------------------------|---------------|--------------------|"
+    } > "$metrics_comparison_file"
+
+    large_diff=false
+
+    # Read CSV
+    while IFS=',' read -r artifact size; do
+      # Skip header
+      [ "$artifact" = "Artifact" ] && continue
+
+      # Trim spaces
+      artifact=$(echo "$artifact" | xargs)
+      size=$(echo "$size" | xargs)
+
+      # Find corresponding artifact size in release file or skip
+      latest_release_size=$(awk -F',' -v art="$artifact" 'NR>1 && $1==art {gsub(/ /,"",$2); print $2}' "$latest_release_metrics_file")
+      [ -z "$latest_release_size" ] && continue
+
+      # Find delta
+      delta=$((size - latest_release_size))
+      abs_delta=${delta#-}
+      percent=$((100 * abs_delta / latest_release_size))
+
+      # Add to file
+      echo "| $artifact | $size | $latest_release_size | $delta | ${percent}% |" >> "$metrics_comparison_file"
+
+      # Check for large diff
+      if [ "$percent" -gt 5 ]; then
+          large_diff=true
+      fi
```

Comment:
Are we no longer using the `significantChangeThresholdPercentage` configured in the plugin? https://github.com/aws/aws-kotlin-repo-tools/blob/56de883c70daaf4389ef5f8d7c5cb6b7a9fa63ac/build-plugins/build-support/src/main/kotlin/aws/sdk/kotlin/gradle/plugins/artifactsizemetrics/ArtifactSizeMetricsPlugin.kt#L103-L106

Let's delete it there if it's no longer being used